exporter/stats/stackdriver/stackdriver_test.go

```diff
@@ -591,6 +591,119 @@ func TestExporter_createMeasure(t *testing.T) {
 	}
 }
 
+func TestExporter_makeReq_withCustomMonitoredResource(t *testing.T) {
+	m, err := stats.NewMeasureFloat64("test-measure", "measure desc", "unit")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer stats.DeleteMeasure(m)
+
+	key, err := tag.NewKey("test_key")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cumView, err := stats.NewView("cumview", "desc", []tag.Key{key}, m, stats.CountAggregation{}, stats.Cumulative{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := stats.RegisterView(cumView); err != nil {
+		t.Fatal(err)
+	}
+	defer cumView.Unregister()
+
+	start := time.Now()
+	end := start.Add(time.Minute)
+	count1 := stats.CountData(10)
+	count2 := stats.CountData(16)
+
+	resource := &monitoredrespb.MonitoredResource{
+		Type:   "gce_instance",
+		Labels: map[string]string{"instance_id": "instance", "zone": "us-west-1a"},
+	}
+
+	tests := []struct {
```

Comment:
Can we also test the case if no custom resource is provided?

Reply:
This should already be covered in the previous tests (TestExporter_makeReq), since they are using the default resource.